stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -290,7 +294,21 @@ public DefaultInspectorModulesBuilder runtimeRepl(RuntimeReplFactory factory) {
      * {@link Context#getDatabasePath} method will be used by default if not overridden here.
      */
     public DefaultInspectorModulesBuilder databaseFiles(DatabaseFilesProvider provider) {
-      mDatabaseFiles = provider;
+      mDatabaseFilesProvider = provider;
+      return this;
+    }
+
+    /**
+     * Extend and provide additional database drivers. Currently two database drivers are supported
+     * in this lib: <br>
+     *   1. <code>SqliteDatabaseDriver</code> - Presents sqlite databases and all tables of the app.<br>
+     *   2. <code>ContentProviderDatabaseDriver</code> - Configure and present content providers data.
+     */
+    public DefaultInspectorModulesBuilder provideDatabaseDriver(Database.DatabaseDriver databaseDriver) {
+      if (mDatabaseDrivers == null) {
+        mDatabaseDrivers = new ArrayList<>();
```

Comment:
Oh, that's why you used the word `provide` here, touché :)

Prefer instead `addDatabaseDriver` then?